src/core/public/ui_settings/ui_settings_service.mock.ts

```diff
@@ -35,6 +36,15 @@ const createSetupContractMock = () => {
     getUpdateErrors$: jest.fn(),
     stop: jest.fn(),
   };
+  setupContract.get$.mockReturnValue(new Rx.Subject());
+  setupContract.getUpdate$.mockReturnValue(
+    new Rx.Subject<{ key: string; newValue: any; oldValue: any }>()
+  );
+  setupContract.getSaved$.mockReturnValue(
+    new Rx.Subject<{ key: string; newValue: any; oldValue: any }>()
+  );
+  setupContract.getUpdateErrors$.mockReturnValue(new Rx.Subject<Error>());
```

Comment:
You should be able to use `any` for the type parameters here without sacrificing any type safety because the type returned by `createSetupContractMock` already has the correct types. If you use `any` then this one fewer places that needs to be updated if a new property is added in the actual implementation.

Reply:
done; changed all parameters to `any`